resources/views/update/results.blade.php

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>Database Migration</title>
+
+        <!-- Fonts -->
+        <link rel="dns-prefetch" href="//fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
```

Comment:
I agree. As far as I know this is only used in the web installer/updater and probably only slipped in, because @ildyria copied & pasted most of the code for the installer/updater from some Laravel cook book. I always have been wondering why the style and layout of the installer/updater looks so different than the rest of Lychee.

When I take care of the installer, I will try to remember to fix at least that. The installer will still have a completely different look & feel than Lychee, because I don't feel appealed to do that, but at least the font can be replaced by "sans-serif" like the rest of Lychee.

Reply:
agreed.

Reply:
As we received the issue https://github.com/LycheeOrg/Lychee/issues/1261 for this in the meanwhile, I am going to close this discussion here as resolved.